cmd/ndt7-client/internal/emitter/interactive_test.go

```diff
@@ -0,0 +1,52 @@
+package emitter
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/m-lab/ndt7-client-go/spec"
+)
+
+// TestInteractiveDownloadError is what happens if you
```

Comment:
You aren't asserting anything

Reply:
Changed the code to have proper unit tests that verify that we emit what we should.